tests/lean/hashmap_on_disk/HashmapMain/Clauses/Template.lean

```diff
@@ -0,0 +1,113 @@
+-- THIS FILE WAS AUTOMATICALLY GENERATED BY AENEAS
```

Comment:
We should have a `Template.lean` generated by Aeneas, and a `Clauses.lean` which is user written and opened by the other files

Reply:
I fixed this: we now need to remove the `sorry` from the `Clauses.lean` files

Reply:
Clauses.lean was missing so I added it for the sake of restoring testing via `lake build` -- agreed that we need to kill the missing proofs in there, but first we need to add reasoning principles for standard library functions, such as checked_sub, and so on